app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new(like_params)
+    @like.user = current_user
+
+    @like.save if @like.valid?
+    redirect_to post_path @like.post
```

Comment:
- [ ] I get this error after creating a like. The like is actually created but your redirection is incorrect. It should be `posts_path(...)`, not `post_path(...)`.

  ![image](https://user-images.githubusercontent.com/54848363/205400118-07b54478-8c30-4986-b6c2-ef2b3ff969ce.png)